esi_leap/common/keystone.py

```diff
@@ -73,3 +73,11 @@ def get_project_name(project_id, project_list=None):
         return project.name if project else ''
     else:
         return ''
+
+
+def get_project_email(project_id):
+    if project_id:
+        project = get_keystone_client().projects.get(project_id)
```

Comment:
Will this return 'None' if project_id is 'None'? If so, the first if statement may not be needed

Reply:
I have simplified the code here